snorkel/analysis/error_analysis.py

```diff
@@ -55,3 +56,59 @@ def get_label_buckets(*y: np.ndarray) -> Dict[Tuple[int, ...], np.ndarray]:
     for i, labels in enumerate(zip(*y_flat)):
         buckets[labels].append(i)
     return {k: np.array(v) for k, v in buckets.items()}
+
+
+def get_label_instances(
+    bucket: Tuple[int, ...], x: np.ndarray, *y: np.ndarray
+) -> np.ndarray:
+    """Return instances in x that were labeled according to bucket.
```

Comment:
Nit: Since 'bucket' isn't a standard term, how about "Return instances in x with the specified combination of labels"

Reply:
Same for the docstring for the return statement